workflows/prognostic_c48_run/tests/test_overrider.py

```diff
@@ -0,0 +1,107 @@
+from datetime import timedelta
+import joblib
+import cftime
+import numpy as np
+import xarray as xr
+
+import fv3gfs.util
+from fv3gfs.util.testing import DummyComm
+
+from runtime.overrider import DatasetCachedByChunk, OverriderAdapter, OverriderConfig
+
+da = xr.DataArray(
+    np.arange(11 * 4 * 6).reshape((11, 4, 6)),
+    dims=["time", "x", "y"],
+    coords={"time": np.arange(2000, 2011)},
+).chunk(
+    {"time": 5, "x": 4, "y": 4}  # type: ignore
+)
+ds = xr.Dataset({"var": da})
+
+
+class MockDerivedState:
```

Comment:
Is there already a mock for this?

Reply:
Not that I am aware of. A search of `MockDerivedState` across fv3net does not find any examples. There is a `MockFV3GFS` in test_derived_state but does not really suit my purposes here.